### ISSUE TITLE
Deployment of rabbitmq on operate first

### ISSUE THREAD

**Issue by tloubrieu-jpl** [operate-first/support#266]
**Is your feature request related to a problem? Please describe.**
I am trying to deploy this application: https://github.com/tloubrieu-jpl/incubator-sdap-nexus/tree/moc_deployment
With command:
    helm install sdap ./helm/  -f values-moc.yaml  -n sdap --dependency-update

The application has a dependency with rabbitmq chart and it fails with error message:
```
Error: roles.rbac.authorization.k8s.io "rabbitmq-endpoint-reader" is forbidden: user "loubrieu@jpl.caltech.edu" (groups=["sdap-mslsp" "system:authenticated:oauth" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:
{APIGroups:[""], Resources:["events"], Verbs:["create"]}

```

**Describe the solution you'd like**
I trust you to propose a solution for that.

**Additional context**
This issue is linked to https://github.com/operate-first/support/issues/206 since spark is also a dependency of this application. To progress on the deployment and not be blocked by spark I temporarily removed the depency on spark. I would need to progress in paralel on both issues.

**Comment by HumairAK** [comment]
@tloubrieu-jpl -- We can get you these permissions, I'll make a pr sometime today for this.

**Comment by tloubrieu-jpl** [comment]
Thanks @HumairAK

**Comment by HumairAK** [comment]
@tloubrieu-jpl you should now be able to create events, this means you can create role/rolebindings with these privileges and provide them to this Serviceaccount

**Comment by tloubrieu-jpl** [comment]
Thanks @HumairAK , I am now bumping into another issue with the rabbitmq deployment.

`create Pod rabbitmq-0 in StatefulSet rabbitmq failed error: pods "rabbitmq-0" is forbidden: unable to validate against any security context constraint: [provider restricted: .spec.securityContext.fsGroup: Invalid value: []int64{1000130001}: 1000130001 is not an allowed group]`

I am trying to override the default fsGroup which I can do, but I have no idea what a valid value is for the fsGroup.

Thanks

**Comment by tumido** [comment]
I've replied in Slack. I've verified it's possible to deploy the helm chart by changing values:

#### Before
```yaml
podSecurityContext:
  enabled: true
  fsGroup: 1001
  runAsUser: 1001
```
#### After
```yaml
podSecurityContext:
  enabled: false
```

Full installation flow:
```
$ curl -O https://raw.githubusercontent.com/bitnami/charts/master/bitnami/rabbitmq/values.yaml

$ vim values.yaml

$ helm repo add bitnami https://charts.bitnami.com/bitnami
"bitnami" has been added to your repositories

$ helm install --generate-name -f values.yaml bitnami/rabbitmq
NAME: rabbitmq-1623923464
LAST DEPLOYED: Thu Jun 17 11:51:08 2021
NAMESPACE: rabbitmq
STATUS: deployed

$ oc expose service rabbitmq-1623923464 --port=http-stats

route.route.openshift.io/rabbitmq-1623923464 exposed

$ oc get all
NAME                        READY   STATUS    RESTARTS   AGE
pod/rabbitmq-1623923464-0   1/1     Running   0          26m

NAME                                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                 AGE
service/rabbitmq-1623923464            ClusterIP   172.30.24.208   <none>        5672/TCP,4369/TCP,25672/TCP,15672/TCP   26m
service/rabbitmq-1623923464-headless   ClusterIP   None            <none>        4369/TCP,5672/TCP,25672/TCP,15672/TCP   26m

NAME                                   READY   AGE
statefulset.apps/rabbitmq-1623923464   1/1     26m

NAME                                           HOST/PORT                                                                   PATH   SERVICES              PORT         TERMINATION   WILDCARD
route.route.openshift.io/rabbitmq-1623923464   rabbitmq-1623923464-rabbitmq.apps.testtcoufal.lab.upshift.rdu2.redhat.com          rabbitmq-1623923464   http-stats                 None

```
![image](https://user-images.githubusercontent.com/7453394/122378264-1a500e00-cf66-11eb-9785-426acbfa5140.png)
